File: challenge6/fizzbuzz.c
#include <stdio.h>

void fizzbuzz(int num);

int main()
{
    for (int i = 0; i <= 100; i++)
    {
        fizzbuzz(i);
        printf("\n");
    }
    return 0;
}

void fizzbuzz(int num)
{
    if (num % 3 == 0 && num % 5 == 0)
    {
        printf("Fizz Buzz");
    } 
    else if (num % 3 == 0)
    {
        printf("Fizz");
    }
    else if (num % 5 == 0)
    {
        printf("Buzz");
    }
    else 
    {
        printf("%i", num);
    }
}

File: challenge6/fizzbuzz.rs
fn main() {
    for n in 1..101 {
        fizzbuzz(n);
    }
}

fn fizzbuzz(n:i32) {
    if n % 3 == 0 && n % 5 == 0 {
        println!("FizzBuzz");
    } else if n % 3 == 0 {
        println!("Fizz");
    } else if n % 5 == 0 {
        println!("Buzz");
    } else {
        println!("{}", n);
    }
}